ruuvitag_sensor/data_formats.py

```diff
@@ -92,6 +92,10 @@ def convert_data(raw: str) -> DataFormatAndRawSensorData:  # noqa: PLR0911
 
         log.debug("Found candidate %s", candidate)
 
+        # Ruuvi Air (E1)
+        if candidate.startswith("FF990406"):
```

Comment:
I think extended might have different value (FF9904E1) based on the documentation?
https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-e1